### PR TITLE
Pos Control logging

### PR DESCRIPTION
**Pull request by lthall**
This PR updates the Pos Control Logging to include the desired values and transform them to NED frame.

### CODE REVIEW

File: libraries/AP_Logger/LogStructure.h [hunk 1]
```diff
@@ -1188,34 +1188,41 @@ struct PACKED log_PSCD {
 // @Field: Vcc: Voltage to Motor
 // @Field: Temp: Motor temperature
 
-// @LoggerMessage: PSC
-// @Description: Position Control data
+// @LoggerMessage: PSCN
+// @Description: Position Control N-axis
```
Comment (rmackay9) [inline libraries/AP_Logger/LogStructure.h:1192]: Let's say "North" instead of "N-axis".

File: libraries/AP_Logger/LogStructure.h [hunk 2]
```diff
@@ -1188,34 +1188,41 @@ struct PACKED log_PSCD {
...
+// @Field: PN: Position relative to EKF origin
+// @Field: DVN: Desired velocity N-axis
+// @Field: TVN: Target velocity N-axis
+// @Field: VN: Velocity N-axis
+// @Field: DAN: Desired acceleration N-axis
+// @Field: TAN: Target acceleration N-axis
+// @Field: AN: Acceleration N-axis
+
+// @LoggerMessage: PSCE
+// @Description: Position Control E-axis
```
Comment (rmackay9) [inline libraries/AP_Logger/LogStructure.h:1204]: and let's replace "E-axis" with "East"

File: libraries/AP_Logger/LogStructure.h [hunk 3]
```diff
@@ -1188,34 +1188,41 @@ struct PACKED log_PSCD {
...
+// @Field: TPE: Target position relative to EKF origin
+// @Field: PE: Position relative to EKF origin
+// @Field: DVE: Desired velocity E-axis
+// @Field: TVE: Target velocity E-axis
+// @Field: VE: Velocity E-axis
+// @Field: DAE: Desired acceleration E-axis
+// @Field: TAE: Target acceleration E-axis
+// @Field: AE: Acceleration E-axis
+
+// @LoggerMessage: PSCD
```
Comment (rmackay9) [inline libraries/AP_Logger/LogStructure.h:1215]: and "D-axis" with "Down"

File: libraries/AP_Logger/LogFile.cpp [hunk 4]
```diff
@@ -553,41 +553,53 @@ void AP_Logger::Write_Winch(bool healthy, bool thread_end, bool moving, bool clu
...
-        vel_desired_z   : vel_desired_z * 0.01f,
-        vel_target_z    : vel_target_z * 0.01f,
-        vel_z           : vel_z * 0.01f,
-        accel_desired_z : accel_desired_z * 0.01f,
-        accel_target_z  : accel_target_z * 0.01f,
-        accel_z         : accel_z * 0.01f,
-        throttle_out    : throttle_out
+    const struct log_PSCE pkt{
+        LOG_PACKET_HEADER_INIT(LOG_PSCE_MSG),
+            time_us         : AP_HAL::micros64(),
```
Comment (rmackay9) [inline libraries/AP_Logger/LogFile.cpp:577]: there's a tiny whitespace issue here .. two characters.. not a big deal of course.

**Comment by rmackay9** [comment]
This looks fine to me.  I've put a few comments about ccomments and whitespace but generally ok.  As discussed let's rebase on master and remove the change to the version and then we can merge, Thanks!

**Comment by rmackay9** [comment]
Merged, thanks!

**Comment by IamPete1** [comment]
I was going to suggest using instance logging since all three messages contain the same stuff. Instance 1 = N, 2 =E, 3 = D

**Pull request merged**